spec-runner/vendor/spec/core/string/tr_spec.rb

```diff
@@ -19,6 +19,7 @@
   it "pads to_str with its last char if it is shorter than from_string" do
     "this".tr("this", "x").should == "xxxx"
     "hello".tr("a-z", "A-H.").should == "HE..."
+    "abcd".tr("a-z", "xxx").should == "xxxx"
```

Comment:
ruby-specs come from upstream. I'd consider this source file immutable. Can you back out this change?

If you think we're missing a test, add an integration test for `String` like these for `Array`:

https://github.com/artichoke/artichoke/blob/b99e3c1163381cec8ab505e33c2201dbe67c7ade/artichoke-backend/src/extn/core/array/mod.rs#L418-L440

https://github.com/artichoke/artichoke/blob/b99e3c1163381cec8ab505e33c2201dbe67c7ade/artichoke-backend/src/extn/core/array/array_functional_test.rb